app/decorators/controllers/refinery/admin/pages_controller_decorator.rb

```diff
@@ -1,12 +1,8 @@
-Refinery::Admin::PagesController.class_eval do
-
-  # Add :copywriting_phrases_attributes to page_params for strong parameters.
-  def page_params_with_copywriting_phrases_params
-    phrases_params = params.require(:page).permit(
-      copywriting_phrases_attributes: [:id, :value]
-    )
-    page_params_without_copywriting_phrases_params.merge(phrases_params)
+module RefineryCopywritingControllerDecorator
+  def permitted_page_params
+    params[:page][:copywriting_phrases_attributes]={} if params[:page][:copywriting_phrases_attributes].nil?
```

Comment:
Instead you can do

```ruby
params[:page][:copywriting_phrases_attributes] ||= {}
```

it has the same effect (except that it would override false too)